### PR TITLE
[ADD][product_brand_stock_account] New module

### PR DESCRIPTION
**Pull request by OpenCode**
This module is different from https://github.com/OCA/brand/tree/13.0/brand_stock_account because here we have the brand of the product in the stock valuation data

### CODE REVIEW

**Review by simoneversienti — APPROVED** [review]

**Review by Dambrous — APPROVED** [review]

**Comment by OpenCode** [comment]
@sbejaoui Can you check this one, too. Thank you.

File: product_brand_stock_account/models/stock_valuation_layer.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class StockValuationLayer(models.Model):
+
+    _inherit = "stock.valuation.layer"
+
+    product_brand_id = fields.Many2one(
+        comodel_name="product.brand",
+        related="product_id.product_brand_id",
+        string="Brand",
+        store=True,
```
Comment (sbejaoui) [inline product_brand_stock_account/models/stock_valuation_layer.py:15]: we need a pre_init_hook to initialize this column

**Review by sbejaoui — CHANGES_REQUESTED** [review]

File: product_brand_stock_account/models/stock_valuation_layer.py (hunk 1 above)
Comment (OpenCode) [reply product_brand_stock_account/models/stock_valuation_layer.py:15]: I have performed several tests. When installing the module, if the products already have a brand associated with it, this column is populated when it is created during installation. So the script is not needed.

**Comment by OpenCode** [comment]
@sbejaoui Let me know if everything is OK

**Review by sbejaoui — APPROVED** [review]

**Pull request merged**